site/en/docs/extensions/mv3/intro/mv3-migration/index.md

```diff
@@ -233,255 +262,340 @@ Replace the following:
+workers as of Chrome 87.
 
-## Modifying network requests  {: #modifying-network-requests }
+{% endAside %}
 
-There is a new
-[declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest) for
-network request modification, which provides an alternative for much of the
-[webRequest](/docs/extensions/reference/webRequest) AP's functionality.
+## Modifying network requests  {: #modifying-network-requests }
 
+There is a new [declarativeNetRequest][api-declarativenetrequest] for network request modification,
+which provides an alternative for much of the [webRequest][api-webrequest] API's functionality.
 
 ### When can you use blocking webRequest?  {: #when-use-blocking-webrequest }
 
-The blocking version of the [webRequest](/docs/extensions/reference/webRequest)
-API still exists in Manifest V3 but its use is restricted to force-installed extensions
-only. See Chrome Enterprise policies:
-[ExtensionSettings](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ExtensionSettings),
-[ExtensionInstallForcelist](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ExtensionInstallForcelist).
+The blocking version of the [webRequest][api-webrequest] API still exists in Manifest V3 but its use
+is restricted to force-installed extensions only. See Chrome Enterprise policies:
+[ExtensionSettings][enterprise-settings], [ExtensionInstallForcelist][enterprise-force-list].
 
-All other extensions must now use
-[declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest) for
-network request modification. This moves the actual modification of the network
-request into the Chrome browser: the extension no longer can read the actual
-network request, and in most cases needs no host permissions.
+All other extensions must now use [declarativeNetRequest][api-declarativenetrequest] for network
+request modification. This moves the actual modification of the network request into the Chrome
+browser: the extension no longer can read the actual network request, and in most cases needs no
+host permissions.
+
+{% Aside 'caution' %}
 
-{% Aside %}
 Request redirects and header modifications **do** require the user to grant host permissions.
-{% endAside %}
 
+{% endAside %}
 
 ### How do you use declarativeNetRequest?  {: #how-use-declarativenetrequest }
 
-Instead of reading the request and programmatically altering it, your extension
-specifies a number of rules, which map a set of conditions to corresponding
-actions. See the
-[declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest)
-reference documentation for a more detailed description of rules.
+Instead of reading the request and programmatically altering it, your extension specifies a number
+of [rules][api-declarative-rules], which map a set of conditions to corresponding actions. See the
+[declarativeNetRequest][api-declarativenetrequest] reference documentation for a more detailed
+description of rules.
```

Comment:
I've apparently entered the late night editing phase where I'm just rewriting everything I read. I'm going to try to stop that now.

```suggestion
Instead of reading the request and programmatically altering it, your extension specifies a number
of [rules][api-declarative-rules]. Each rule contains a set of actions to perform when a given set
of conditions are matched. For example, you could define a rule that removes "cookie" headers when a
request is sent to a specific domain. See the [declarativeNetRequest][api-declarativenetrequest]
reference documentation for a more detailed description of rules
```